docs/sources/alerting/manage-notifications/template-notifications/write-templates.md

```diff
@@ -0,0 +1,252 @@
+---
+aliases:
+keywords:
+  - grafana
+  - alerting
+  - notifications
+  - templates
+  - write templates
+title: Write templates
+weight: 400
+---
+
+# Write templates
+
+You write message templates in Go's templating language, [text/template](https://pkg.go.dev/text/template). In the following topics we will look at how to write message templates, and then look at a number of more complex examples that you might want to use as a reference for customizing your notifications.
+
+## Opening and closing tags
```

Comment:
```suggestion
## Open and close tags
```

Reply:
This one needs to keep the `ing` as its a noun